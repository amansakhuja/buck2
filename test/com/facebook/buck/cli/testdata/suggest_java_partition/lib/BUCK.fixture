java_library(
  name = 'lib',
  srcs = glob(['*.java']),
  deps = ['//third-party:third-party'],
  visibility = [
    'PUBLIC',
  ],
)
