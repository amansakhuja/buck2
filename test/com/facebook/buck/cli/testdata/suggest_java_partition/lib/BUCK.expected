java_library(
  name = 'lib',
  exported_deps = [
    ':lib.Alpha',
    ':lib.Bravo',
    ':lib.Charlie',
    ':lib.Delta',
  ],
  visibility = [
    'PUBLIC',
  ],
)

java_library(
  name = 'lib.Alpha',
  srcs = [
    'Alpha.java',
  ],
  deps = [
    ':lib.Bravo',
    ':lib.Charlie',
    '//third-party:third-party',
  ],
  visibility = [
    'PUBLIC',
  ],
)

java_library(
  name = 'lib.Bravo',
  srcs = [
    'Bravo.java',
  ],
  deps = [
    ':lib.Delta',
  ],
  visibility = [
    'PUBLIC',
  ],
)

java_library(
  name = 'lib.Charlie',
  srcs = [
    'Charlie.java',
  ],
  deps = [
    ':lib.Delta',
  ],
  visibility = [
    'PUBLIC',
  ],
)

java_library(
  name = 'lib.Delta',
  srcs = [
    'Delta.java',
  ],
  deps = [
    '//third-party:third-party',
  ],
  visibility = [
    'PUBLIC',
  ],
)
